source/data-api/custom-endpoints.txt

```diff
@@ -293,6 +293,16 @@ An endpoint can require authenticated users to provide additional
 authorization information in the request. You define the authorization
 scheme for each custom endpoint by configuring the endpoint function.
 
+Endpoints natively support a set of built-in authorization schemes
+that use a secret string to prove that the request is authorized. You
+can also define a custom authorization scheme that can replace or
+supplement the built-in schemes.
```

Comment:
i find the phrase "replace or supplement" vague. perhaps:

```suggestion
can also define a custom authorization scheme that
you can use together with or instead of the built-in schemes.
```